style.css

```diff
@@ -0,0 +1,97 @@
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 24px;
+  margin-right: 16px;
+  margin-top: 58px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.logo a {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  align-items: center;
+  text-align: center;
+  display: flex;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  width: 100vw;
+  height: 718px;
+  background-image: url("./Images/header-shapes mobile.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 70px 24px;
+}
+
+.headline h1 {
+  width: 327px;
+  height: 150px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  list-style: 20px;
+  color: #172b4d;
+  display: flex;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline p {
+  width: 327px;
+  height: 100px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.headline h3 {
+  color: #7f8cff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 20px;
+}
+
+.social-media-icons {
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you correct the margin from the bottom of "LETS CONNECT" to your social media icons

| Your Design  | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-27 10-40-42](https://user-images.githubusercontent.com/52934128/170675685-4da93345-a09d-4faf-9f0c-854dbb554fcc.png)  | ![Screenshot from 2022-05-27 10-41-04](https://user-images.githubusercontent.com/52934128/170675730-149eccba-a705-4301-93ba-67c4e2ea68af.png)  |

Reply:
Thank you so much